### DISCUSSION TITLE
Is notify-manager still using the batch function provided by react in react-query by default?

### DISCUSSION THREAD

**Discussion by 87xie** [TanStack/query#7529] [Q&A] [answered]
Hello,

I noticed the comment in the [notifyManager.ts](https://github.com/TanStack/query/blob/main/packages/query-core/src/notifyManager.ts) file:

```ts
/**
 * Use this method to set a custom function to batch notifications together into a single tick.
 * By default, React Query will use the batch function provided by ReactDOM or React Native.
 */
const setBatchNotifyFunction = (fn: BatchNotifyFunction) => {
  batchNotifyFn = fn;
}
```

In React Query v4, the batch function is set by default in the [setBatchUpdatesFn.ts](https://github.com/TanStack/query/blob/v4/packages/react-query/src/setBatchUpdatesFn.ts) file:

```ts
import { notifyManager } from '@tanstack/query-core'
import { unstable_batchedUpdates } from './reactBatchedUpdates'

notifyManager.setBatchNotifyFunction(unstable_batchedUpdates)
```

This is no longer the case in v5, am I correct? If so, should we update the comment for better clarity?

If I'm missing something obvious, please let me know. 🙏

**Comment by TkDodo** [comment, marked as answer]
yeah the comment is outdated. We're not using that function at all, so maybe we can even deprecate it and remove it later? The docs say that we use it for solid, but I can't find any usages. @ardeora is this still used for the solid adapter?

```ts
import { notifyManager } from '@tanstack/query-core'
import { batch } from 'solid-js'

notifyManager.setBatchNotifyFunction(batch)
```
